services/ui-src/src/verbiage/errors.ts

```diff
@@ -21,14 +21,14 @@ export const validationErrors = {
   INVALID_RATIO: "Response must be a valid ratio",
 };
 
+const genericErrorContent =
+  "Something went wrong on our end. Refresh your screen and try again. <br/>If this persists, contact the MDCT Help Desk with questions or to request technical assistance by emailing ";
+
 export const reportErrors = {
-  GET_REPORT_DATA_FAILED:
-    "Report data could not be loaded. Please contact support.",
-  SET_REPORT_DATA_FAILED:
-    "Report data could not be saved. Please contact support.",
-  GET_REPORT_FAILED: "Report could not be loaded. Please contact support.",
-  GET_REPORTS_BY_STATE_FAILED:
-    "Reports could not be loaded. Please contact support.",
-  SET_REPORT_FAILED: "Report could not be updated. Please contact support.",
-  DELETE_REPORT_FAILED: "Report could not be deleted. Please contact support.",
+  GET_REPORT_DATA_FAILED: genericErrorContent,
+  SET_REPORT_DATA_FAILED: genericErrorContent,
+  GET_REPORT_FAILED: genericErrorContent,
+  GET_REPORTS_BY_STATE_FAILED: genericErrorContent,
+  SET_REPORT_FAILED: genericErrorContent,
+  DELETE_REPORT_FAILED: genericErrorContent,
```

Comment:
+1 to not replacing these all with generic error. There is definitely room for improvement on error messaging here, but at the very least differentiating these error messages is helpful for developer experience and debugging!

Reply:
Just reiterating that I think we should try to preserve some of the custom error text, specifically

- `GET_REPORT_FAILED`
- `GET_REPORTS_BY_STATE_FAILED`
- `SET_REPORT_FAILED`

Those are the ones that are being set by various functions in the ReportProvider. Could we sync up with design/product to create versions of the new error (with link, etc) for each of those use cases? It might be as easy as keeping the first sentence for each of these, and the rest of the description can be the same as the generic error. The issue is that if we make all of the error messages the same, we eliminate our first entry point for debugging, which is to determine which function generated the error we see in the UI.